src/current/_includes/metrics-usage/tenant.consumption.sql_pods_cpu_seconds.md

```diff
@@ -0,0 +1,5 @@
+The number of RUs that were consumed by CPU usage of SQL processes (not storage processes). The CPU seconds is converted to Request Units using this equivalency:
```

Comment:
'sql processes' could link to https://www.cockroachlabs.com/docs/v23.2/architecture/sql-layer

'storage processes' could link to https://www.cockroachlabs.com/docs/v23.2/architecture/storage-layer